web/src/utilities/utilities.js

```diff
@@ -58,7 +58,16 @@ export function getBuildVersion() {
 }
 
 export function parseIconUri(uri) {
-  const splitUri = uri.split("?")
+  const splitUri = uri.split("?");
+  if (!splitUri.length) {
```

Comment:
this check is redundant because split returns an array with at least 1 element.  There is no guarantee that there will be 2 elements.